packages/commonwealth/client/scripts/views/components/react_quill_editor/react_quill_editor.tsx

```diff
@@ -234,6 +234,7 @@ const ReactQuillEditor = ({
               handleToggleMarkdown={handleToggleMarkdown}
               setIsPreviewVisible={setIsPreviewVisible}
               isDisabled={isDisabled}
+              content={getTextFromDelta(contentDelta)}
```

Comment:
you can pass just a bool down like so

`isPreviewDisabled={content?.length < 1}`

Then in `toolbar.tsx` you will add only one line 
`disabled={isPreviewDisabled}`

Reply:
Addressed.